### PR TITLE
update dxDiagram demo description

### PR DESCRIPTION
**Pull request by LanaDX**

### CODE REVIEW

File: JSDemos/Demos/Diagram/Containers/description.md [hunk 1]
```diff
@@ -4,4 +4,6 @@ This demo shows the Diagram component's capability to wrap shapes into container
 * a container is dragged with its content,
 * a container can be collapsed with a collapse button.
```
Comment (albertov05) [inline JSDemos/Demos/Diagram/Containers/description.md:6]: Can you please make the following change to line 3 (above)?
`* a container can contain other shapes (including other containers),`

Please capitalize "A" at the beginning, and change the comma at the end to a period:

`* A container can contain other shapes (including other containers).`

File: JSDemos/Demos/Diagram/Containers/description.md [hunk 2]
```diff
@@ -4,4 +4,6 @@ This demo shows the Diagram component's capability to wrap shapes into container
 * a container is dragged with its content,
```
Comment (albertov05) [inline JSDemos/Demos/Diagram/Containers/description.md]:
```suggestion
* A container is dragged with its content.
```

File: JSDemos/Demos/Diagram/Containers/description.md [hunk 3]
```diff
@@ -4,4 +4,6 @@ This demo shows the Diagram component's capability to wrap shapes into container
...
 * a container can be collapsed with a collapse button.
 
-To wrap a shape, just drop it into a container.
\ No newline at end of file
+To wrap a shape, just drop it into a container.
```
Comment (albertov05) [inline JSDemos/Demos/Diagram/Containers/description.md]:
```suggestion
To wrap a shape, drop it into a container.
```

File: JSDemos/Demos/Diagram/Containers/description.md (hunk 1 above)
Comment (LanaDX) [reply JSDemos/Demos/Diagram/Containers/description.md:6]: fixed

Comment (albertov05) [reply JSDemos/Demos/Diagram/Containers/description.md:6]: Thanks!

**Review by AnnaKondratova — APPROVED** [review]

**Pull request merged**